website/docs/constructs/library/generated/_consumption-opensearch.mdx

```diff
@@ -49,14 +49,18 @@ Default configuration
   <TabItem value="typescript" label="TypeScript" default>
 
   ```typescript
-const osCluster = new dsf.consumption.OpenSearchCluster(this, 'MyOpenSearchCluster',{
-  domainName:"mycluster",
-  samlEntityId:'<IdpIdentityId>',
-  samlMetadataContent:'<IdpMetadataXml>',
-  samlMasterBackendRole:'<IAMIdentityCenterAdminGroupId>',
-  deployInVpc:true,
-  removalPolicy:cdk.RemovalPolicy.DESTROY
-});
+    const osCluster = new dsf.consumption.OpenSearchCluster(this, 'MyOpenSearchCluster',{
```

Comment:
Can you also add a documentation page for the API, call it `Opensearch API - Bring your own Opensearch cluster`.